README.md

```diff
@@ -161,6 +161,13 @@ The IPinfo library can be authenticated with your IPinfo API token, which is pas
 
 In-memory caching of `details` data is provided by default via the [cachetools](https://cachetools.readthedocs.io/en/latest/) library. This uses an LRU (least recently used) cache with a TTL (time to live) by default. This means that values will be cached for the specified duration; if the cache's max size is reached, cache values will be invalidated as necessary, starting with the oldest cached value.
 
+You can access the cached property to see if the response was cached.
```

Comment:
Two nits here:

1. Add a newline between the codeblock and this line.
2. Convert the word "cached" here to an inline codeblock, i.e. "You can access the `cached` property..." This clearly distinguishes it as referring to something in the code.

Reply:
I accidentally clicked resolve. I'm working on this.